go-app-ussd_nurse_rapidpro.js

```diff
@@ -2,33 +2,110 @@ var go = {};
 go;
 
 go.app = function() {
+    var _ = require('lodash');
     var vumigo = require('vumigo_v02');
+    var Q = require('q');
     var MenuState = vumigo.states.MenuState;
     var ChoiceState = vumigo.states.ChoiceState;
     var PaginatedChoiceState = vumigo.states.PaginatedChoiceState;
     var FreeText = vumigo.states.FreeText;
     var EndState = vumigo.states.EndState;
     var Choice = vumigo.states.Choice;
     var SeedJsboxUtils = require('seed-jsbox-utils');
-    var IdentityStore = SeedJsboxUtils.IdentityStore;
     var JsonApi = vumigo.http.api.JsonApi;
+    var utils = SeedJsboxUtils.utils;
     var App = vumigo.App;
 
     var GoNDOH = App.extend(function(self) {
         App.call(self, 'state_start');
-        var utils = SeedJsboxUtils.utils; //replace with utils not in Seed
 
         //variables for services
-        var is;
+        /*var is;*/
 
         self.init = function() {
             // initialise services
             //replace identity store
-            is = new IdentityStore(
+            /*is = new IdentityStore(
```

Comment:
Can we remove this instead of commenting it out?

Reply:
ahh this too thanks